### PR TITLE
💀 😈 🎃

### PR DESCRIPTION
**Pull request by Hacksore**
RFC for the best moji for commit `#666`

### CODE REVIEW

**Comment by maxdemaio** [comment]
# 😈

**Comment by MVAodhan** [comment]
LGTM

**Comment by Hacksore** [comment]
we added `<Suspense>` in commit number 666 
https://github.com/typehero/typehero/commit/e5a0a4bc81fd09063636b594768b05ea7cc20afd

File: .depcheckrc.yaml [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-  # misc
-  - '@monaco-editor/loader'
-  - '@t3-oss/env-nextjs'
-  - 'postcss'
-  - 'raw-loader'
-  - 'tsx'
-  - 'postcss'
-  - 'autoprefixer'
-  - 'prettier-plugin-tailwindcss'
-  # some big g shit right here
```
[the remark this answers is not in the document]
Comment (Hacksore) [reply .depcheckrc.yaml:19]: image

**Pull request closed without merging**